string Class       # Name of the detected class
float32 probability # Detection confidence (0.0 to 1.0)
int32 xmin         # Top-left x-coordinate of the bounding box
int32 ymin         # Top-left y-coordinate of the bounding box
int32 xmax         # Bottom-right x-coordinate of the bounding box
int32 ymax         # Bottom-right y-coordinate of the bounding box
int32 id           # Unique identifier for the bounding box